xml/System.Runtime.Intrinsics/Vector128.xml

```diff
@@ -53,12 +53,14 @@
         <Parameter Name="vector" Type="System.Runtime.Intrinsics.Vector128&lt;T&gt;" RefType="this" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <typeparam name="U">To be added.</typeparam>
-        <param name="vector">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of the input vector.</typeparam>
+        <typeparam name="U">The type of the vector <paramref name="vector" /> should be reinterpreted as.</typeparam>
+        <param name="vector">The vector to reinterpret.</param>
+        <summary>Reinterprets a <see cref="T:System.Runtime.Intrinsics.Vector128`1" /> as a new <see cref="T:System.Runtime.Intrinsics.Vector128`1" />.</summary>
```

Comment:
I'm not sure this is an improvement, `vector` is not a clear differentation when you also have `Vector64<T>` and `Vector256<T>` in the `System.Runtime.Intrinsics` namespace and `Vector<T>`, `Vector2`, `Vector3`, and `Vector4` in the `System.Numerics` namespace.

All of these types will be frequently used together and we already have some methods for switching between them (such as going from `Vector128<T>` to `Vector256<T>`) and may have more in the future (such as going from `Vector4` to `Vector128<float>`).

I think it would be good to determine if there is a way to keep the original comments and have them rendered correctly (as that was indicating both the vector type, `Vector128`, and the specific generic instantiation that it filled, such as `<Byte>`).

Reply:
I agree. `vector` is ambiguous

Reply:
@tannergooding  @carolEidt I learned that the build process that converts triple slash comments to xml is not converting them correctly. Instead of keeping the {keyword} sections from the comments, it's converting them to "`1".

I think for this case, we will have to manually copy and paste the triple slash comments to the PR.

Reply:
> is not converting them correctly.

This is likely dependent on what the language spec considers "correct" here and may require a language or compiler issue to resolve.

It is also liekly dependent on how the api-docs transforms will handle it when rendering.

Reply:
Talked to Tanner in person. I know exactly what he would like to see, so I'll make the modifications and add a new iteration to the PR soon.